### PR TITLE
Cleanup nitpick suggestion in the lint command

### PR DESCRIPTION
**Pull request by andytom**
**What this PR does / why we need it**:
Have fixed up a non-blocking nitpick change from #6310, this just
switches us to use a regular string rather than a string.Builder for the
summary.

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: cmd/helm/lint.go [hunk 1]
```diff
@@ -91,12 +91,11 @@ func newLintCmd(out io.Writer) *cobra.Command {
 
 			fmt.Fprintf(out, message.String())
 
-			var summary strings.Builder
-			fmt.Fprintf(&summary, "%d chart(s) linted, %d chart(s) failed", len(paths), failed)
+			summary := fmt.Sprintf(&summary, "%d chart(s) linted, %d chart(s) failed", len(paths), failed)
```
Comment (bacongobbler) [inline cmd/helm/lint.go]: correct usage would be `summary := fmt.Sprintf("%d chart(s) linted[....]`.

Comment (andytom) [reply cmd/helm/lint.go]: Doh, I knew I should have double checked before I pushed. Have fixed this now.

File: cmd/helm/lint.go [hunk 2]
```diff
@@ -91,12 +91,11 @@ func newLintCmd(out io.Writer) *cobra.Command {
...
 
-			var summary strings.Builder
-			fmt.Fprintf(&summary, "%d chart(s) linted, %d chart(s) failed", len(paths), failed)
+			summary := fmt.Sprintf("%d chart(s) linted, %d chart(s) failed", len(paths), failed)
 			if failed > 0 {
-				return errors.New(summary.String())
+				return errors.New(summary)
 			}
-			fmt.Fprintf(out, "%s\n", summary.String())
+			fmt.Fprintf(out, "%s\n", summary)
```
Comment (bacongobbler) [inline cmd/helm/lint.go]: nit: you could just use fmt.Fprintln here instead.

**Review by bacongobbler — APPROVED** [review]
One small nitpicking comment but functionality LGTM! Let me know if you'd like to address that comment first before merging or not. Thanks

**Comment by andytom** [comment]
Since this is a fix for a nitpick in another PR I'll fix this first.

File: cmd/helm/lint.go (hunk 2 above)
Comment (andytom) [reply cmd/helm/lint.go]: @bacongobbler fixed

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**